odd-platform-api/src/main/java/org/opendatadiscovery/oddplatform/repository/reactive/ReactiveMetadataFieldValueRepositoryImpl.java

```diff
@@ -0,0 +1,76 @@
+package org.opendatadiscovery.oddplatform.repository.reactive;
+
+import java.util.Collection;
+import java.util.List;
+import lombok.RequiredArgsConstructor;
+import org.jooq.impl.DSL;
+import org.opendatadiscovery.oddplatform.dto.metadata.MetadataOrigin;
+import org.opendatadiscovery.oddplatform.model.tables.pojos.MetadataFieldValuePojo;
+import org.opendatadiscovery.oddplatform.model.tables.records.MetadataFieldValueRecord;
+import org.opendatadiscovery.oddplatform.repository.util.JooqReactiveOperations;
+import org.springframework.stereotype.Repository;
+import reactor.core.publisher.Flux;
+import reactor.core.publisher.Mono;
+
+import static org.opendatadiscovery.oddplatform.model.Tables.METADATA_FIELD;
+import static org.opendatadiscovery.oddplatform.model.Tables.METADATA_FIELD_VALUE;
+
+@Repository
+@RequiredArgsConstructor
+public class ReactiveMetadataFieldValueRepositoryImpl implements ReactiveMetadataFieldValueRepository {
+    private final JooqReactiveOperations jooqReactiveOperations;
+
+    @Override
+    public Flux<MetadataFieldValuePojo> bulkCreate(final Collection<MetadataFieldValuePojo> pojos) {
+        if (pojos.isEmpty()) {
+            return Flux.just();
+        }
+        final List<MetadataFieldValueRecord> records = pojos.stream()
```

Comment:
Minor: this piece of code is duplicated with ReactiveAbstractCRUDSmthRepository. Maybe it makes sense to extract it somewhere to jooqQueryHelper or smth?